roles/maintenance_31_nginx/tasks/main.yml

```
@@ -93,27 +93,22 @@
     valid_at:
       one_month_later: "+1m"
   register: nginx_cert_info
-  loop: "{% set cert_content = [] %}{% for result in nginx_cert_files_content.results %}{% for data in ([(result.content | b64decode | community.crypto.split_pem)] | flatten) %}{% set cert_content = cert_content.append(data) %}{% endfor %}{% endfor %}{{ cert_content }}"
+  loop: >-
+     {%- set cert_content = [] -%}
+     {%- for result in nginx_cert_files_content.results -%}
+     {%- set cert_content = cert_content + ([(result.content | b64decode | community.crypto.split_pem)] | flatten) -%}
+     {%- endfor -%}
+     {{- cert_content -}}
   changed_when: false
   ignore_errors: true
   delegate_to: localhost
 
-- <<: *task
-  vars:
-    taskid: 31-014
-    name: "SSL_ If in use, will SSL certificate be valid in a month"
-  ansible.builtin.set_fact:
-    nginx_cert_expires_soon: true
-  loop: "{{ nginx_cert_info.results }}"
-  when: not (item.valid_at.one_month_later | default(true))
-  loop_control:
-    loop_var: item
-  changed_when: false
-
 - <<: *task
   vars:
     taskid: 31-014
     name: "SSL_ If in use, will SSL certificate be valid in a month"
```

One minor typo, then we're good to go:

```suggestion
    name: "SSL: If in use, will SSL certificate be valid in a month"
```